test/ipld-dag-cbor.js

```diff
@@ -115,10 +115,11 @@ module.exports = (repo) => {
       })
 
       it('resolver.get just CID', (done) => {
-        resolver.get(cid1, (err, result) => {
+        resolver.get(cid1, (err, results) => {
```

Comment:
In all other tests you didn't rename `result` to `results`. This is just a something I'd like to mention in case you care a lot about consistency. I'm happy to leave it like that (there's no point of wasting time on such small things, but sometimes I do then though ;)